packages/cli/src/commands/add.js

```diff
@@ -20,7 +20,7 @@ const register = program => program
 
 async function action(contractNames, options) {
   ConfigVariablesInitializer.initStaticConfiguration()
-  if(!options.skipCompile) await compile()
+  if(!options.skipCompile) await Compiler.call()
```

Comment:
Have you thought about a way of centralizing this `Compiler.call()` call so we don't call it in each command `action` function?

Reply:
Added an issue for that :) https://github.com/zeppelinos/zos/issues/508